airflow/providers/cncf/kubernetes/executors/kubernetes_executor_utils.py

```diff
@@ -165,6 +163,14 @@ def _run(
 
         last_resource_version: str | None = None
 
+        # For info about k8s timeout settings see
+        # https://github.com/kubernetes-client/python/blob/94e42113a1fe5c580917decacdde879eab7406b3/examples/watch/timeout-settings.md
+        # and https://github.com/kubernetes-client/python/blob/b47caad922709350f477210317ac7f9574a72a97/kubernetes/client/api_client.py#L336-L339
```

Comment:
Why are these permanent links pointing to different versions?

Reply:
🤷